src/app/app.component.css

```diff
@@ -0,0 +1,25 @@
+.app {
```

Comment:
Try to beautify application a bit. At least layout need to be a bit more stable.